crt_portal/static/sass/custom/typography.scss

```diff
@@ -47,7 +47,8 @@ h3,
 }
 
 .h3__display {
-  @include text--heading__medium();
+  font-size: $theme-heading-font-size-md;
+  line-height: 1.5;
```

Comment:
@cantsin The change in typograpghy.scss is needed to adhere to the design audit mobile header 3. 
![image](https://user-images.githubusercontent.com/66343959/91731370-c7db6780-eb74-11ea-80a6-3da7f0f07cfd.png)
Since they are the same on desktop and mobile I wanted to not use the media query inside the text--heading__medium. Based on your comment I moved the attributes into the mixin as that conforms to typography.scss _display style classes. However I wanted to bring up a small point that the text--heading__medium mixin where the change now resides is used quite heavily throughout our application. So maybe keeping this change strictly in .h3__display might be the better course of action. For what its worth I did not encounter any glaring issues on mobile going through landing and reporting pages that used this text--heading__medium. Let me know your thoughts on this  issue.

Regarding the use of mixins I think you have a great idea that I hadn't initially thought about. My only reservation is that my understanding of this card was that it would mostly be applied to the landing page. If we create mixins do we keep them inside landing.scss or should we broaden our scope. This point also leads me to another thought regarding the general upkeep of sass/css.